frontend/app/events/eventDetail/eventDetail.component.ts

```diff
@@ -0,0 +1,46 @@
+import { Component, OnInit } from "@angular/core";
+import { ActivatedRoute, Params } from "@angular/router";
+
+import { Event, EventService } from "./../";
+
+@Component({
+    moduleId: module.id,
+    selector: "cg-event-detail-page",
+    templateUrl: "eventDetail.component.html",
+})
+
+export class EventDetailComponent implements OnInit {
+    private activeEvent: Event;
+    private errorMessage: string;
+    constructor(private eventService: EventService, private route: ActivatedRoute) {
+        this.activeEvent = new Event();
+    }
+
+    public ngOnInit(): void {
+        this.route.params.forEach((params: Params) => {
+          let id = + params["id"];
+          this.getEventDetails(id);
+        });
+    }
+
+    public clearError() {
+        this.errorMessage = "";
+    }
+
+    private getEventDetails(id: number) {
+        this.eventService.getEvent(id)
+                        .subscribe(
+                            event => this.setActiveEvent(event),
+                            error => this.setErrorMessage(<any> error),
+                        );
+    }
+
+    private setErrorMessage(errorMessage: string) {
+        this.errorMessage = errorMessage;
+    }
+
+    private setActiveEvent(event: Event) {
+        this.activeEvent = event;
+        this.eventService.setActiveEvent(event);
```

Comment:
Du holst dir in der `getEventDetails` das Event aus dem Service und dann schreibst du ihn wieder zurück. Wäre es nicht besser wenn sich der Service einfach das Event merkt?